apps/transport/lib/jobs/create_tokens_job.ex

```diff
@@ -35,8 +38,45 @@ defmodule Transport.Jobs.CreateTokensJob do
     end)
   end
 
+  # - Create tokens for contacts without an organization
+  # - Set this token as the default token
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: %{"action" => "create_tokens_for_contacts_without_org"}}) do
+    contact_ids_default_token =
```

Comment:
Plus facile à comprendre pour moi si on écrit ça :

```suggestion
    contact_ids_with_a_default_token =
```